documentation/PullRequestChecklist.md

```diff
@@ -0,0 +1,44 @@
+<!--===- documentation/PullRequestChecklist.md 
+
+   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+   See https://llvm.org/LICENSE.txt for license information.
+   SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+-->
+
+# Pull request checklist
+Please review the following items before submitting a pull request.  This list
+can also be used when reviewing pull requests.
+*  Verify that new files have a license with correct file name.
+*  Run `git diff` on all modified files to look for spurious changes such as
+   ```#include <iostream>```.
+*  If you added code that causes the compiler to emit a new error message, make
+   sure that you also added a test that causes that error message to appear
+   and verifies its correctness.
+*  Annotate the code and tests with appropriate references to constraint and
+   requirement numbers from the Fortran standard.
+*  Check all dereferences of non-owning pointers.
```

Comment:
How about "Check dereferences of pointers and optionals when necessary."? Owning vs. non-owning isn't relevant.

Reply:
Much better.  I'll change it.